### ISSUE TITLE
_Span object as no attribute 'get_context'

### ISSUE THREAD

**Issue by Shackelford-Arden** [honeycombio/opentelemetry-exporter-python#9]
Hello!

Starting to explore the world of OpenTelemetry and trying to get some stuff figured out, ended up hitting a snag and would appreciate some insight. Currently attempting to figure out how to instrument with OT, but ship off to HoneyComb.

If I'm in the wrong place, please let me know :) Happy to take this to wherever.

# A Simplified App

I have the following app in Python that utilizes FastAPI:

```python
from fastapi import FastAPI
from opentelemetry import trace
from opentelemetry.ext.honeycomb import HoneycombSpanExporter
from opentelemetry.instrumentation.fastapi import FastAPIInstrumentor
from opentelemetry.sdk.trace import TracerProvider
from opentelemetry.sdk.trace.export import BatchExportSpanProcessor

from myapp.models.settings import settings

exporter = HoneycombSpanExporter(
    service_name=settings.ot_service_name,
    writekey=settings.honeycombio_api_key,
    dataset=settings.ot_data_set
)

span_processor = BatchExportSpanProcessor(exporter)
trace.get_tracer_provider().add_span_processor(span_processor)

app = FastAPI(title="Super Awesome App")

@app.get("/health")
async def system_health():
    return "Healthy!"

FastAPIInstrumentor.instrument_app(app)

```

# My Understanding (likely flawed!)

Based on the reading I've done so far, my understanding is that with the combination of the FastAPI Instrumentor and the HoneycombSpanExporter, some basic level metrics from FastAPI would be kicked off and shipped off to HC.

# Errors Received

With the above, I receive the following error when I hit the `/health` endpoint:

```python
Exception while exporting Span batch.
Traceback (most recent call last):
  File "../lib64/python3.8/site-packages/opentelemetry/sdk/trace/export/__init__.py", line 311, in _export_batch
    self.span_exporter.export(self.spans_list[:idx])  # type: ignore
  File "../lib64/python3.8/site-packages/opentelemetry/ext/honeycomb/__init__.py", line 70, in export
    hny_data = _translate_to_hny(spans)
  File "../lib64/python3.8/site-packages/opentelemetry/ext/honeycomb/__init__.py", line 86, in _translate_to_hny
    ctx = span.get_context()
AttributeError: '_Span' object has no attribute 'get_context'
```

**Comment by Shackelford-Arden** [comment]
Thought I'd come and leave a note here that this occurs with or without the specific FastAPI instrumentation included.

**Comment by lizthegrey** [comment]
we've just released v0.15b0, please let us know if your issue is solved.
